sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_policies.py

```diff
@@ -81,3 +81,28 @@ def _encode(self, url):
             algorithm="HS256",
         )
         return six.ensure_str(encoded)
+
+
+class ApiManagementProxy(SansIOHTTPPolicy):
+
+    def __init__(self, endpoint, proxy_endpoint):
+        # type: (str, str) -> None
+        """Create a new instance of the policy.
+
+        :param endpoint: endpoint to be replaced
+        :type endpoint: str
+        :param proxy_endpoint: proxy endpoint
+        :type proxy_endpoint: str
+        """
+        self._endpoint = endpoint
+        self._proxy_endpoint = proxy_endpoint
+
+    def on_request(self, request):
+        # type: (PipelineRequest) -> typing.Union[None, typing.Awaitable[None]]
```

Comment:
I don't think this return typing is correct, we're not returning an awaitable here. I'd just type it as `# type: (PipelineRequest) -> None`